README.md

```diff
@@ -118,7 +118,7 @@ See [GOVERNANCE](docs/GOVERNANCE.md).
 ## Contributing
 
 If you want to build the project, check [docs/INSTALL.md](docs/INSTALL.md). If you want to help the project,
-see [docs/CONTRIBUTING.md](docs/CONTRIBUTING.md) and read everything in the [docs folder](docs/) of the repository.
```

Comment:
It is move, not removed.

https://github.com/organicmaps/organicmaps/blob/70b03b770385c90987db95ae42824b6cdb0e8dc0/docs/CONTRIBUTING.md#contributing mentions several way of contributing, not only coding (and probably more could be mentioned, especially OSM mapping)

Alternative would be mentioning donations, mapping and other noncode help directly in https://github.com/organicmaps/organicmaps#contributing

Or, I guess, it can be assumed that anyone on Github is interested in help by making PRs and donations/code/mapping is irrelevant.